app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -59,7 +75,8 @@ angular.module('openshiftConsole')
           if ($scope.createPersistentVolumeClaimForm.$valid) {
             $scope.disableInputs = true;
             var claim = generatePersistentVolumeClaim();
-            DataService.create('persistentvolumeclaims', null, claim, context)
+            var createClaimVersion = APIService.objectToResourceGroupVersion(claim);
```

Comment:
I'd use the `createPVCVersion` from above since we already have it

Reply:
Yeah doesn't really make sense to do the work again, should still have the same `rgv` value.  ok will update.

Reply:
Going to move it up as well, most of these I have near the top of the file, before the `ProjectService.get()`